bam2bcf_indel.c

```diff
@@ -439,10 +439,12 @@ int bcf_call_gap_prep(int n, int *n_plp, bam_pileup1_t **plp, int pos, bcf_calla
     }
     free(ref2); free(query);
     { // compute indelQ
-        int *sc, tmp, *sumq;
-        sc   = alloca(n_types * sizeof(int));
-        sumq = alloca(n_types * sizeof(int));
-        memset(sumq, 0, sizeof(int) * n_types);
+        int sc_a[16], sumq_a[16];
+        int tmp, *sc = sc_a, *sumq = sumq_a;
+        if (n_types > 1) {
+            sc   = malloc(n_types * sizeof(int));
+            sumq = calloc(n_types,  sizeof(int));
```

Comment:
Samtools doesn't build with -Wc++-compat, so maybe it's not worth changing unless purely for consistencies sake. (This is time limited anyway?)